preorder.html

```diff
@@ -0,0 +1,164 @@
+          </p>
+        </div>
+        <table id="preorder">
+         <tr>
+           <th></th>
+           <th>STANDARD</th>
+           <th>LARGE</th>
+           <th>MEGA</th>
+         </tr>
+          <tr>
+            <td>Resource Units *</td>
+            <td>200 <small>GBrh</small></td>
+            <td>400 <small>GBrh</small></td>
+            <td>600 <small>GBrh</small></td>
+          </tr>
+          <tr>
+            <td>Storage</td>
+            <td>5 <small>GB</small></td>
+            <td>30 <small>GB</small></td>
+            <td>100 <small>GB</small></td>
+          </tr>
+          <tr>
+            <td>Bandwidth</td>
+            <td>Unlimited</td>
+            <td>Unlimited</td>
+            <td>Unlimited</td>
+          </tr>
+          <tr class="beta">
+            <td>Beta Access</td>
+            <td>FREE</td>
+            <td>FREE</td>
+            <td>FREE</td>
+          </tr>
+          <tr class="price">
+            <td>Pre-order Special</td>
+            <td>$6 <br /><small>/ month</small></td>
+            <td>$12 <br /><small>/ month</small></td>
+            <td>$24 <br /><small>/ month</small></td>
+          </tr>
+          <tr>
+            <td></td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-standard-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                  data-key="pk_test_euywwlSEioegnYyeZPPjYGZ2"
+                  data-name="Sandstorm"
+                  data-image="/images/sandstorm-purplecircle.png"
+                  data-description="Pre-order Standard Instance"
+                  data-amount="600"
+                  data-label="Standard >>"
+                  data-allow-remember-me="false"
+                  data-panel-label="{{amount}} / Month">
+                </script>
+              </form>
+            </td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-large-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                  data-key="pk_test_euywwlSEioegnYyeZPPjYGZ2"
+                  data-name="Sandstorm"
+                  data-image="/images/sandstorm-purplecircle.png"
+                  data-description="Pre-order Large Instance"
+                  data-amount="1200"
+                  data-label="Large >>"
+                  data-allow-remember-me="false"
+                  data-panel-label="{{amount}} / Month">
+                </script>
+              </form>
+            </td>
+            <td>
+              <form class="preorder" action="https://payments.sandstorm.io/pre-order-mega-instance" method="POST">
+                <script
+                  src="https://checkout.stripe.com/checkout.js" class="stripe-button"
```

Comment:
I wonder if the script actually needs to be sourced 3 times. More likely it searches for all tags with class `stripe-button`, so really only the last one needs to include the `src` (or you could have a separate <script> tag with the `src` later on, but no other attributes). If that works then let's do that so we aren't executing redundant Javascript.